tests/e2e/local.test.ts

```diff
@@ -1,8 +1,8 @@
 import { test, expect } from "./fixtures";
 
-test("can analyze example.com", async ({ page, extensionId }) => {
+test("localhost works", async ({ page, extensionId }) => {
 	// Goto example.com and open the extension popup
-	await page.goto("https://example.com/");
+	await page.goto("localhost:5173/local");
```

Comment:
This only works if the port is available, and if it is not, there is no recourse. We should probably create an environment variable that sets the server port.